extension/scripts/features/travel-tab-title-timer/ttTravelTabTitleTimer.ts

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+(async () => {
+	if (!isFlying()) return;
+
+	featureManager.registerFeature(
+		"Tab Title Timer",
+		"travel",
+		() => settings.pages.travel.tabTitleTimer,
+		null,
+		showTabTimer,
+		removeTabTimer,
+		{
+			storage: ["settings.pages.travel.tabTitleTimer"],
+		},
+		null
+	);
+
+	let timer_update_interval = undefined;
+
+	async function showTabTimer() {
+		timer_update_interval = setInterval(() => {
+			updateTabTimer();
+		}, 1000);
+
+
+		updateTabTimer();
+	}
+
+	function updateTabTimer() {
+
+		// Steal text from existing HTML element and set it as the doccument title
+		let time = (document.querySelector("[class*='progressTextLineBreaker__'] time") as HTMLElement).innerText;
```

Comment:
Just assuming this element is present doesn't work. In my case the feature completely doesn't work after a refresh because of this. Since this runs on a 1 second time you can just early return if the element doesn't return as it will resolve itself after 1 second.